Subject: uitests: force internal snapshots in test_snapshot.py
From: Cole Robinson crobinso@redhat.com Wed Aug 7 11:38:26 2024 -0400
Date: Mon Aug 12 19:55:58 2024 -0400:
Git: 0cad0c4dff6aa2c5e46a286a0cacfea124fb48bf

Incase the libvirt test driver ever starts advertising support

Signed-off-by: Cole Robinson <crobinso@redhat.com>

diff --git a/tests/uitests/test_snapshot.py b/tests/uitests/test_snapshot.py
index 4eb478cf..72ece0be 100644
--- a/tests/uitests/test_snapshot.py
+++ b/tests/uitests/test_snapshot.py
@@ -80,6 +80,7 @@ def testSnapshotLifecycle(app):
     newwin = app.find_window("Create snapshot")
     snapname = "testnewsnap"
     newwin.find("Name:", "text").set_text(snapname)
+    newwin.find("internal", "radio button").click()
     newwin.find("Description:", "text").set_text("testdesc")
     newwin.find("Finish", "push button").click()
     lib.utils.check(lambda: not newwin.showing)
@@ -99,6 +100,7 @@ def testSnapshotLifecycle(app):
     newwin = app.find_window("Create snapshot")
     snapname = "testnewsnap"
     newwin.find("Name:", "text").set_text(snapname)
+    newwin.find("internal", "radio button").click()
     newwin.find("Finish", "push button").click()
     lib.utils.check(lambda: not newwin.showing)
     newc = win.find(snapname, "table cell")
@@ -126,6 +128,7 @@ def testSnapshotMisc1(app):
     newwin = app.find_window("Create snapshot")
     snapname1 = "testnewsnap1"
     newwin.find("Name:", "text").set_text(snapname1)
+    newwin.find("internal", "radio button").click()
     newwin.find("Finish", "push button").click()
     lib.utils.check(lambda: not newwin.showing)
     newc = win.find(snapname1, "table cell")
@@ -136,6 +139,7 @@ def testSnapshotMisc1(app):
     lib.utils.check(lambda: not vmrun.sensitive)
     win.find("snapshot-add", "push button").click()
     newwin = app.find_window("Create snapshot")
+    newwin.find("internal", "radio button").click()
     # Force validation error
     newwin.find("Name:", "text").set_text("bad name")
     newwin.find("Finish", "push button").click()
